Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginUIFactory.swift

```diff
@@ -50,16 +50,19 @@ public struct SimplifiedLoginUIFactory {
         let nc = UINavigationController()
         nc.pushViewController(s, animated: false)
         
+        let url = URL(string: viewModel.localisation.privacyPolicyURL)!
+        let webVC = WebViewController()
+        webVC.loadURL(url)
```

Comment:
Line is 64 already calls loadURL. Is both of them needed?

Reply:
Yes and this is why. User:
1.  opens privacy policy and browses through the pages 
2.  goes back to simplified login main view 
3. opens privacy policy again (should be privacy policy first page - not the last which he saw)

WkWebView caches content so there is no performance issues

Reply:
Ok I see. But if we only keep loadURL in line 64? Then wont this give us the same behaviour as you mention? (i.e. first page always opening) :)
Or maybe I am missing something 🤔

Reply:
I think that now we gain on loading it previously 🤔

Reply:
What is the gain. Is it a performance gain?

Reply:
If we are loading it earlier, should be faster I think

Reply:
Yes, there is a little faster for the first time

Reply:
I removed the first loadURL call :)

Reply:
Do you know if the WKWebView caches?

Reply:
If so then it might be good to have the first loadUrl?

Reply:
let's leave it as it is and loadURL only on demand :) sounds more natural

Reply:
still looks pretty good

Reply:
Ok 👍